Kickstarter-iOS/DataSources/CommentRepliesDataSource.swift

```diff
@@ -0,0 +1,28 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class CommentRepliesDataSource: ValueCellDataSource {
+  internal enum Section: Int {
+    case comment
+    case replies
+  }
+
+  internal func load(comment: Comment) {
```

Comment:
That's a really good point, because unlike the `CommentsViewModel`, we can have two outputs - one for the context called `loadComment` and another just for the replies called `loadCommentReplies` and in the data source we can have `load(comment: Comment)` and `loadReplies(comments: [Comment]`. That way we are never dependent on a network request to load the initial comment.

So I'll rename it to `createContext(comment: Comment)`?

Reply:
I'm worse with naming, so what ever you name it  😂 .